.devcontainer/devcontainer.json

```diff
@@ -12,9 +12,15 @@
 		"--cap-add=SYS_PTRACE",
 		"--security-opt=seccomp:unconfined",
 		"--security-opt=apparmor:unconfined",
-		"--volume=/tmp/.X11-unix:/tmp/.X11-unix"
+		"--volume=/tmp/.X11-unix:/tmp/.X11-unix",
+		"--gpus" "all",
+		"--rm"
```

Comment:
Why --rm?

It's actually convenient for the container to remain on your system so it doesn't need to be re-built every time you re-open code.

Reply:
Used that for testing to make sure I always have a clean docker. Removed -rm.